migrations/default/2_create_triggers/up.sql

```diff
@@ -0,0 +1,380 @@
+--
+-- handle crash record insert by copying
+-- to the vz_ and unified tables
+--
+create or replace function db.crashes_cris_insert_rows()
```

Comment:
i like it! any suggestions?

Reply:
Is there a particular column you're asking about in terms of suggestions? I was responding in general about the use of prefixes, but if it's helpful, I think it's best to lean into the concept of store every piece of data or knowledge just once, so maybe:

* database name denotes the application: `visionzero`
* schema denotes the logical group of table like objects and their supporting objects: `vz`, `cris`, `public`, and maybe `vz_lookup`, `cris_lookup` if we want to go that route
* tables -
  * uniformly have a `id` column which identifies any one record from any other and
  * and every column is named with plain, whole words like `secondary_road_name` for example
    * and only if there is a compelling reason, denote groups of columns based on a trait based with a suffix or prefix, such as `fatality_count` in the `public.crashes` table maybe getting a suffix like `fatality_count_computed` or in the `public.people` table a column like `is_primaryperson`, where `is_` is a prefix that denotes a boolean value
      * A counter example of something I'd like to avoid is `_fl` for a boolean value because, 1, it's not clear what a `fl` is unless you have domain specific knowledge, and 2, it's a boolean typed value, and the typing of the column does a plenty good job of denoting that it's a boolean. For the above point, `primaryperson` as a boolean column would be fine, without the `is_` but `is_` is plain enough language that it works for me too.

I hope this helpful, and if i'm off on a tangent, please disregard!

Reply:
I came back to add -- what I've proposed above is very much based on my personal preferences, and I don't want to make any claim that these are "industry best practices" or anything like that. They are Frank's practices, but they have worked well for me in the past. 

The industry best practice would be simple: 
* Above all, and with no exception, be **_consistent_**.